src/utils.js

```diff
@@ -381,7 +381,7 @@ function findSuccessorTab(state, tab, exclude) {
   }
 
   // Previously active tab
-  if (state.activateAfterClosing === 'prev_act') {
+  if (state.actTabs != null && state.activateAfterClosing === 'prev_act') {
```

Comment:
And here we just add the actTabs check: `if (state.actTabs && state.activateAfterClosing === 'prev_act') {`